src/App.tsx

```diff
@@ -5,15 +5,16 @@ import { Helmet } from 'react-helmet';
 // Import antd stylesheets
 import 'antd/dist/antd.css';
 import './App.less';
-import Home from './containers/home/Home';
+// import Home from './containers/home/Home';
```

Comment:
Any reason to keep this import as a comment?

Reply:
Nope